Makefile

```
@@ -29,18 +29,16 @@ override PG_CXXFLAGS += -std=c++17 -Wno-sign-compare -Wno-register ${DUCKDB_BUIL
 
 SHLIB_LINK += -Wl,-rpath,$(PG_LIB)/ -lpq -Lthird_party/duckdb/build/$(DUCKDB_BUILD_TYPE)/src -L$(PG_LIB) -lduckdb -lstdc++ -llz4
 
-COMPILE.cc.bc = $(CXX) -Wno-ignored-attributes -Wno-register $(BITCODE_CXXFLAGS) $(CXXFLAGS) $(PG_CPPFLAGS) $(PG_CXXFLAGS) -I$(INCLUDEDIR_SERVER) -emit-llvm -c
-
-%.bc : %.cpp
-	$(COMPILE.cc.bc) $(SHLIB_LINK) -I$(INCLUDE_SERVER) -o $@ $<
-
 include Makefile.global
 
 # We need the DuckDB header files to build the .o files. We depend on the
 # duckdb Makefile, because that target pulls in the submodule which includes
 # those header files.
 $(OBJS): third_party/duckdb/Makefile
 
+COMPILE.cc.bc += $(PG_CPPFLAGS)
+COMPILE.cxx.bc += $(PG_CXXFLAGS)
```

I'm guessing this still needs the PG_CPPFLAGS too.
```suggestion
COMPILE.cxx.bc += $(PG_CXXFLAGS) $(PG_CPPFLAGS)
```

---

doesn't seem so! everything looking good on 22.04, 24.04, and locally on MacOS.